java/src/test/java/com/saucelabs/simplesauce/EdgeTest.java

```diff
@@ -1,72 +1,63 @@
 package com.saucelabs.simplesauce;
 
 import org.junit.Test;
-import org.mockito.Mockito;
 
 import static org.junit.Assert.assertEquals;
 
 public class EdgeTest extends BaseConfigurationTest {
     @Test
     public void withEdge_default() {
```

Comment:
@titusfortner This test seems different than what previously existed. The previous test set the sauceOptions, started a new session, which means that it actually created a full RemoteWebDriver object, and then asserted that the object was constructed correctly.
The test in this PR seems to only check that the property was set correctly. Which doesn't seem nearly as valuable as it doesn't guarantee the correct creation of the RemoteWebDriver that will be created to interact with Sauce.

Reply:
So, there are three pieces now that get checked.
1. Options class state was set correctly
2. Options class conversion happens correctly
3. Session class uses conversion

Separation of concerns means keeping the unit tests for these three distinct activities independent.